conf/messages.en

```diff
@@ -6,8 +6,9 @@ site.delete.hidden = Remove {0}
 site.edit.hidden = Change {0}
 site.no = No
 site.yes = Yes
-site.continue = Save and continue
-site.submit = Continue
+site.continue = Continue
+site.saveAndContinue = Save and continue
+site.submit = Submit
```

Comment:
Is this key needed?

Reply:
Wasn't sure whether to leave it in as it may be used later but have removed it for now